src/scripts/main.js

```diff
@@ -1 +1,15 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const listImage = document.querySelector('ul');
+
+listImage.addEventListener('click', (item) => {
+  item.preventDefault();
+
+  let targetEl = item.target;
+
+  if (targetEl.tagName === 'IMG') {
+    targetEl = targetEl.parentElement;
+  }
+  largeImg.src = targetEl.href;
```

Comment:
The code assumes that `targetEl` has an `href` attribute after checking if the tag is 'IMG'. This assumption may not hold true if the parent element is not an anchor (`<a>`). Consider adding a check to ensure `targetEl` is an anchor element before accessing `href`.